projects/UWMadison_Wright.yaml

```
@@ -1,11 +1,11 @@
 Name: UWMadison_Wright
-Description: Project for Daniel Wright hydroclimate extremes research group from UW-Madison
+Description: Project for Daniel Wright's research group at UW-Madison
 Department: Civil and Environmental Engineering
-FieldOfScience: Atmospheric Sciences
+FieldOfScience: Civil Engineering
 Organization: University of Wisconsin-Madison
 PIName: Daniel Wright
 
-ID: '698'
+ID: '705'
```

```suggestion
ID: '698'
```

Revert the change in project ID